cmd/kubeadm/app/util/runtime/runtime.go

```diff
@@ -151,20 +151,28 @@ func (runtime *DockerRuntime) RemoveContainers(containers []string) error {
 
 // PullImage pulls the image
 func (runtime *CRIRuntime) PullImage(image string) error {
-	out, err := runtime.exec.Command("crictl", "-r", runtime.criSocket, "pull", image).CombinedOutput()
-	if err != nil {
-		return errors.Wrapf(err, "output: %s, error", string(out))
+	var err error
+	var out []byte
+	for i := 0; i < constants.PullImageRetry; i++ {
+		out, err = runtime.exec.Command("crictl", "-r", runtime.criSocket, "pull", image).CombinedOutput()
```

Comment:
Ignoring errors here doesn't look good to me.

Reply:
it's not ignoring, it retries until it succeeds if it doesn't it logs the error. We can still log the error on each retry but I'm not sure if it's needed